src/components/Advertisement.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+import { Col, Card, CardBody } from 'reactstrap';
+
+const TCG = () => {
+  return (
+    <Col xs="12">
+      <div className="centered pt-2">
+        <iframe
+          title="tcg"
+          className="advertisement"
+          src="https://content.tcg20life.com/tcgafa2.html?pk=CubeCobra&categoryId=1&campaign=affiliate&source=CubeCobra&medium=CubeCobra"
+          id="tcg-a1"
+          width="728"
+          height="90"
+          frameBorder="0"
+        />
+      </div>
+    </Col>
+  );
+};
+
+const Advertisment = () => {
+  const [option] = useState(Math.floor(Math.random() * 4));
+
+  if (option === 0) {
```

Comment:
Considered moving these options into their own object? Instead of duplicating the component code, we could simplify a bit:

```
  const ads = [
    {
      text: "Cube Cobra's hosting fees are covered completely by donations. If you enjoy the service Cube Cobe provides, please consider",
      link: {
        text: "donating",
        href: "https://www.patreon.com/cubecobra"
      }
    },
     ...
 ]
if (shownAdvertisement <= 1) {
    return (
      <Col x="12" className="py-2">
        <Card>
          <CardBody>
            {ads[shownAdvertisement].text}
            <strong>
              <a href={ads[shownAdvertisement].link.href}> {ads[shownAdvertisement].link.text}</a>
            </strong>
          </CardBody>
        </Card>
      </Col>
    )
  } else {
    return <TCG />;
  }
```